### PR TITLE
Added Openshift 3.11 and k8s 1.12.2 with multus providers.

### PR DESCRIPTION
**Pull request by SchSeba**
Add providers for multus:
* openshift 3.11
* k8s 1.12.2

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirtci/52)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 * `k8s-1.11.0` k8s-1.11.0 cluster based on the centos7 image, provisioned with kubeadm
 * `k8s-multus-1.10.4:`: k8s-1.10.4 cluster based on the centos7 image and uses multus CNI, provisioned with kubeadm
 * `k8s-multus-1.11.1:`: k8s-1.11.1 cluster based on the centos7 image and uses multus CNI, provisioned with kubeadm
+* `k8s-multus-1.11.1:`: k8s-1.12.2 cluster based on the centos7 image and uses multus CNI, provisioned with kubeadm
```
Comment (phoracek) [inline README.md]: item title says 1.11.1, but description says 1.12.2

File: README.md [hunk 2]
```diff
@@ -17,6 +18,7 @@
 * `os-3.10-multus` os-3.10 cluster with multus cni support based on the centos7 image, provisioned with openshift-ansible
 * `os-3.11.0` os-3.11.0 cluster based on the centos7 image, provisioned with openshift-ansible
 * `os-3.11-crio` os-3.11 cluster with CRI-O support based on the centos7 image, provisioned
+* `os-3.10-multus` os-3.11 cluster with multus cni support based on the centos7 image, provisioned with openshift-ansible
```
Comment (phoracek) [inline README.md]: that should be 3.11

File: k8s-multus/scripts/provision.sh [hunk 3]
```diff
@@ -144,6 +144,35 @@ networking:
   podSubnet: 10.244.0.0/16
 EOF
 
+# New configuration for kubernetes > 1.12
```
Comment (phoracek) [inline k8s-multus/scripts/provision.sh]: Should be >= 1.12

File: manifests/flannel.yaml [hunk 4]
```diff
@@ -98,8 +98,7 @@ spec:
       nodeSelector:
         beta.kubernetes.io/arch: amd64
       tolerations:
-      - key: node-role.kubernetes.io/master
-        operator: Exists
+      - operator: Exists
```
Comment (phoracek) [inline manifests/flannel.yaml]: How does this work without a key? What does it do

**Comment by cynepco3hahue** [comment]
@SchSeba Can you please reply to @phoracek comments and rebase your PR.

**Comment by SchSeba** [comment]
@cynepco3hahue done lets wait for https://github.com/kubevirt/kubevirt/pull/1726 to finish

File: README.md [hunk 5]
```diff
@@ -28,15 +30,17 @@
 * **Deprecated**: `kubevirtci/os-3.9.0-crio:`: `sha256:107d03dad4da6957e28774b121a45e177f31d7b4ad43c6eab7b24d467e59e213`
 * **Deprecated**: `kubevirtci/os-3.10.0:`: `sha256:cc418c0c837d8e6c9a31a063762d9e4c8bfc70a1fcca10823b11c6d8a7ae2394`
 * **Deprecated**: `kubevirtci/os-3.10.0-crio:`: `sha256:56debd7bc2ce87dd616ebc30f06971e388b6983c0cda8646a7563e1dafadb69b`
-* `kubevirtci/os-3.10.0-multus`: `sha256:875c973099141ab2013aaf51ec2b35b5326be943ef1437e4a87e041405e724ca`
+* **Deprecated**: `kubevirtci/os-3.10.0-multus`: `sha256:875c973099141ab2013aaf51ec2b35b5326be943ef1437e4a87e041405e724ca`
```
Comment (rmohr) [inline README.md]: This is great, that means that I just need to rebase for #49 and don't have to enable the status subresource feature for openshift 3.10 there.

File: manifests/flannel.yaml (hunk 4 above)
Comment (rmohr) [reply manifests/flannel.yaml]: Looks good.

**Pull request merged**